hopo/ueberblick_gremien.tex

```diff
@@ -40,56 +25,47 @@ \subsection{Aus der jüngsten Geschichte der Mitbestimmung an den Hochschulen}
+Angeführt wird eine Fakultät durch eine Dekanin, sowie den Studiendekaninnen und Prodekaninnen als Fakultätsvorstand. 
+Unsere Studiengänge gehören zu den Fakultäten für Physik und Astronomie sowie Mathematik und Informatik, wobei Prof. Schultz-Coulon in der ersteren und Prof. Bastian in der letzteren Dekan ist. 
+Die 12 Fakultäten der Universität Heidelberg sind weiter aufgegliedert in Institute und Seminare, welche wiederum in fachspezifischere Arbeitsgruppen unterteilt sind. 
 
-\subsection{Hintergründe zur VS}
+\subsection{Senat und universitätsweite Gremien}
+Da sich auch die verschiedenen Fakultäten untereinander absprechen müssen, gibt es zentrale universitätsweite Gremien, wobei die Struktur ähnlich wie auf Fakultätsebene ist. 
+Die Professorinnen haben eine deutliche Mehrheit, denn neben den 16 gewählten Professorinnen und nur 4 Studentinnen sind auch noch der Rektor, die vier Prorektorinnen, die Kanzlerin sowie alle 12 Dekaninnen Senatsmitglieder. 
+Hier herrscht also kaum studentische Mitgestaltungsmöglichkeit. 
+Jedoch hat der Senat diverse Unterausschüsse, insbesondere den Senatsausschuss für Lehre (SAL), welcher sich ähnlich wie die Studienkommissionen alleine mit den Themen Studium und Lehre befasst, diesmal nur mit Themen, die alle Fakultäten betreffen. 
+Zum Beispiel wurde lange über Anwesenheitspflichten und den Master of Education gesprochen, sodass nicht jede Fakultät eine eigene Regelung treffen muss. 
+Neben dem Bestätigen und Durchwinken von Entscheidungen der Fakultäten, beschäftigt sich der Senat hauptsächlich mit strategischen und richtungsweisenden Entscheidungen der Universität. 
+Diese betreffen hauptsächlich professorale Angelegenheiten der Forschung und eher weniger, aber umso wichtigere studentische Belange. 
 
-\paragraph{Mitgliedschaft in der Verfassten Studierendenschaft}
+Der Rektor der Universität ist zurzeit Bernhard Eitel. 
+Als Leiter des Rektorats stellt er mit diesem die Hochschulleitung dar und vertritt die Hochschule nach innen und außen, wie es so schön heißt. 
+Die vier Prorektorinnen sind jeweils für bestimmte Bereiche zuständig, unter anderem gibt es den Bereich Studium und Lehre. 
 
-Alle Studierenden sind qua Immatrikulation automatisch Mitglied der Verfassten Studierendenschaft. Weil sie die einzige Möglichkeit zur Partizipation an legitimierter Studierendenvertretung ist und ihre Leistungen für alle Studierenden offen stehen müssen, ist ein Austritt nicht möglich.
+Die organisatorischen Aufgaben der Universität werden von der (Zentralen) Universitätsverwaltung (ZUV) übernommen, welche von der Kanzlerin Angela Kalous geleitet wird und in acht Dezernate gegliedert ist. 
```

Comment:
Wie oft ändert sich das? ggfalls namen in einen zu-aktualisieren-befehl auslagern.

Reply:
Kanzlerin ist glaube ich auch auf 6 Jahre gewählt